goes-plume-viewer/src/components/mapLayer/index.jsx

```diff
@@ -12,9 +12,12 @@ export const MapLayer = ({ plume, handleLayerClick, plumeId, hoveredPlumeId, set
         const rasterSourceId = getSourceId("raster"+plumeId);
         const rasterLayerId = getLayerId("raster"+plumeId);
         const polygonSourceId = getSourceId("polygon"+plumeId);
-        const polygonLayerId = getLayerId("polygon"+plumeId);
+        const polygonLayerId = getLayerId("polygon" + plumeId);
+        const rasterApiUrl = config?.rasterApiUrl
+            ? config.rasterApiUrl
+            : process.env.REACT_APP_RASTER_API_URL;
 
-        addSourceLayerToMap(map, feature, rasterSourceId, rasterLayerId);
+        addSourceLayerToMap(map, feature, rasterSourceId, rasterLayerId,rasterApiUrl);
```

Comment:
using context api instead of the props drilling would minimize the changes necessary